config/pipelines/high_throughput_heron_384_tailed.yml

```diff
@@ -0,0 +1,26 @@
+---
+Heron-384 Tailed A: # Heron 384-well pipeline specific to PCR 1 plate
```

Comment:
I feel like I might have asked this before, but do you know why it's got that word 'tailed' in there? What does it mean?

Reply:
It's something about the way the adapters work. I think its a case of the tags/adapters being built in to the primer-panel oligos themselves.